src/MapfileStyleParser.spec.ts

```diff
@@ -64,7 +70,55 @@ describe('MapfileStyleParser implements StyleParser', () => {
       expect(geoStylerStyle).toBeDefined();
       expect(geoStylerStyle).toEqual(raster_simple_raster);
     });
+    /*
+    it('can read a simple MapFile RasterSymbolizer with many classes intervals', async () => {
+      expect.assertions(2);
+      const mapfile = fs.readFileSync( './data/mapfiles/raster_simple_raster_many_classes_filter_intervals.map', 'utf8');
+      const geoStylerStyle = await styleParser.readStyle(mapfile);
+      expect(geoStylerStyle).toBeDefined();
+      expect(geoStylerStyle).toEqual(raster_simple_raster_many_classes_filter_intervals);
+    });
```

Comment:
If tests are commented, you can merge, but please, don't close the jira ticket. Or create a new one to keep a track of the work to do.
Thus, the best would be to enable these tests before to merge.

Reply:
OK no problem. With Balthasar, we thought it was nice to comment the tests that are still not implemented in the parser so that it's possible to uncomment and implement them one by one...